internal/decision/engine_test.go

```diff
@@ -1199,24 +1352,24 @@ func TestTaggingUseful(t *testing.T) {
 	}
 }
 
-func partnerWantBlocks(e *Engine, keys []string, partner peer.ID) {
+func partnerWantBlocks(e *Engine, wantBlocks []string, partner peer.ID) {
 	add := message.New(false)
-	for i, letter := range keys {
+	for i, letter := range wantBlocks {
 		block := blocks.NewBlock([]byte(letter))
-		add.AddEntry(block.Cid(), int32(len(keys)-i), pb.Message_Wantlist_Block, true)
+		add.AddEntry(block.Cid(), int32(len(wantBlocks)-i), pb.Message_Wantlist_Block, true)
 	}
 	e.MessageReceived(context.Background(), partner, add)
 }
 
-func partnerWantBlocksHaves(e *Engine, keys []string, wantHaves []string, sendDontHave bool, partner peer.ID) {
+func partnerWantBlocksHaves(e *Engine, wantBlocks []string, wantHaves []string, sendDontHave bool, partner peer.ID) {
```

Comment:
rationale: I didn't understand this call at first so I made the parameters name symmetric, `wantHave` <-> `wantBlocks`.
Which lead me to rename the parameter in the other function above (since it was probably a copy and paste).